schedule/functional/extra_tests_gnome.yaml

```diff
@@ -20,20 +20,20 @@ conditional_schedule:
                 - x11/keyboard_layout_gdm
             'x86_64':
                 - x11/keyboard_layout_gdm
-    steam:
+    x86_64_tests:
         ARCH:
             'x86_64':
                 - x11/steam
+                - x11/wine
+                - x11/chrome                
```

Comment:
You seem to have some trailing whitespaces that make the testsuite fail.

Reply:
Fixed.